### PR TITLE
BasicSearch - closed #27

### PR DESCRIPTION
**Pull request by ksdurg**
Added a BasicSearchViewModel with 3 basic variables (first name, last… name, position) and a BasicSearch view under the Coaches views folder.

### CODE REVIEW

**Comment by Daniel-Watkins** [comment]
I noticed that when you moved your BasicSearchViewModel.cs to the SearchViewModels folder you didn't change the namespace. I recommend changing it just for the sake of consistency.

What it should be:      namespace gybitg.Models.SearchViewModels

This means you have to change the view as well from:
@using gybitg.ViewModels;
@model gybitg.ViewModels.BasicSearchViewModel

to:
@using gybitg.Models.SearchViewModels;
@model gybitg.Models.SearchViewModels.BasicSearchViewModel

I pulled down the branch and did this on my end to make sure it didn't break anything and it all worked.

File: gybitg/Views/Manage/ManageNavPages.cs [hunk 1]
```diff
@@ -25,15 +25,18 @@ public static class ManageNavPages
...
         public static string SearchResults => "SearchResults";
 
         public static string AdvancedSearch => "AdvancedSearch";
 
         public static string TwoFactorAuthentication => "TwoFactorAuthentication";
 
-        public static string IndexNavClass(ViewContext viewContext) => PageNavClass(viewContext, Index);
+        //public static string BasicSearchClass(ViewContext viewContext) => BasicSearchClass(viewContext, BasicSearch);
 
-        public static string AdvancedSearchNavClass(ViewContext viewcontext) => PageNavClass(viewcontext, AdvancedSearch);
```
Comment (Daniel-Watkins) [inline gybitg/Views/Manage/ManageNavPages.cs:36]: Did you get Alex's permission to delete his AdvancedSearchNavClass and why is yours commented out?

Comment (ksdurg) [reply gybitg/Views/Manage/ManageNavPages.cs:36]: Not sure.  I didn't delete anyone's code.  And I didn't comment out any of my own code lines.

Comment (ksdurg) [reply gybitg/Views/Manage/ManageNavPages.cs:36]: Only thing I deleted was my own BasicSearchViewModel.cs

Comment (Daniel-Watkins) [reply gybitg/Views/Manage/ManageNavPages.cs:36]: Could you add it back in and uncomment your NavClass before merging it.

**Review by Daniel-Watkins — APPROVED** [review]
Looks good to me.

**Review by AlexKSparks — APPROVED** [review]
Looks good!

**Review by keithrutler — APPROVED** [review]
Looks good to me

**Pull request merged**